addons/mail/static/tests/emoji/emoji_tests.js

```diff
@@ -97,8 +97,8 @@ QUnit.test("recent category (basic)", async () => {
     await contains(".o-EmojiPicker-navbar [title='Frequently used']");
     await contains(".o-Emoji", {
         text: "😀",
-        after: ["span", { text: "Frequently used" }],
-        before: ["span", { text: "Smileys & Emotion" }],
+        after: ["span", { textContent: "Frequently used" }],
+        before: ["span", { textContent: "Smileys & Emotion" }],
```

Comment:
Is there any issue with `text:`?

Reply:
The `span` is not specific enough in this case, so there are multiple matches (nested `span` that contains the same text, but the top level one also contains the emoji, so using `textContent` only keeps the nested exact match).